src/views/Peers.tsx

```diff
@@ -154,99 +181,141 @@ export const Peers = () => {
         });
     }
 
+    const setUpdateGroupsVisible = (peerToAction:Peer, status:boolean) => {
+        if (status) {
+            dispatch(peerActions.setPeer({...peerToAction}))
+            dispatch(peerActions.setUpdateGroupsVisible(true))
+            return
+        }
+        dispatch(peerActions.setPeer(null))
+        dispatch(peerActions.setUpdateGroupsVisible(false))
+    }
+
+    const renderPopoverGroups = (label: string, groups:Group[] | string[] | null, peerToAction:PeerDataTable) => {
+        const content = groups?.map((g,i) => {
+            const _g = g as Group
+            const peersCount = ` - ${_g.PeersCount || 0} ${(_g.PeersCount && parseInt(_g.PeersCount) > 1) ? 'peers' : 'peer'} `
+            return (
+                <div key={i}>
+                    <Tag
+                        color="blue"
+                        style={{ marginRight: 3 }}
+                    >
+                        <strong>{_g.Name}</strong>
+                    </Tag>
+                    <span style={{fontSize: ".85em"}}>{peersCount}</span>
```

Comment:
Codacy has a fix for the issue: [Replace `····················<span·style={{fontSize:·".85em"` with `··········<span·style={{·fontSize:·".85em"·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
          <span style={{ fontSize: ".85em" }}>{peersCount}</span>
```